components/powersensor/server_test.go

```diff
@@ -0,0 +1,140 @@
+package powersensor_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	pb "go.viam.com/api/component/powersensor/v1"
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/components/powersensor"
+	"go.viam.com/rdk/components/sensor"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/testutils/inject"
+)
+
+var (
+	workingPowerSensorName = "ps1"
+	failingPowerSensorName = "ps2"
+	missingPowerSensorName = "ps3"
```

Comment:
Mild preference for the names to be semantically meaningful. If I have an unexpected problem on "ps2", I don't want to need to go look up that this is the one that's supposed to be failing.